README.md

```diff
@@ -177,6 +177,24 @@ final ListenableFuture<B> future = getFuture();
 FuturesExtra.addFailureCallback(future, System.out::println);
 ```
 
+#### Concurrency limiting
+
+If you want to fire of a large number of asynchronous requests or jobs, it can be useful to limit how many will run concurrently.
+To help with this, there is a new class called `ConcurrencyLimiter`. You use it like this:
+
+```java
+final int maxConcurrency = 10;
+final ConcurrencyLimiter<T> = ConcurrencyLimiter.create(maxConcurrency);
+for (int i = 0; i < 1000; i++) {
+  final ListenableFuture<T> future = concurrencyLimiter.add(() -> createFuture());
+}
+```
+The concurrency limiter will ensure that at most 10 futures are created and incomplete at the same time. All the jobs that
```

Comment:
Line length seems a bit random, max 80 chars makes the raw text file more readable, I think.